processor/configuration/src/main/resources/schemas/actions/kafka_topic_sink_0.1.json

```diff
@@ -7,9 +7,28 @@
       "title": "Topic Name",
       "description": "The topic where to send the event.",
       "example": "my-topic"
+    },
+    "kafka_broker_url": {
```

Comment:
@lucamolteni Should we enforce a `pattern` here to ensure that the user can only provide a `*.bf2.kafka.rhcloud.com:443` broker url. At the minute the user can still target any URL for the broker URL.

Reply:
@robpblake I've created a follow up ticket https://issues.redhat.com/browse/MGDOBR-887